README.md

```diff
@@ -22,3 +22,111 @@ This is a beta version until audited by the security team. Any comments or sugge
+
+## Live monitoring tool
+
+The live monitoring tool enables real time monitoring or a given set of Bridge related events and methods.
+
+This tool can be helpful to track peg-ins, peg-outs or any other specific bridge action in real time.
+
+It can be executed against a node running in any network (regtest, testnet or mainnet).
+
+To print in the console all the transactions sent to the bridge with their method and events data, the following command can be used:
+
+> node tool/live-monitor.js
+
+The tool will start checking block by block (starting from the latest one), transaction by transaction, filtering the transaction sent to the bridge and printing a payload like this:
+
+```js
+...
+
+{
+  txHash: '0xda3db1d8b949e99a4f0fb27dc867981e79b457339b8df94a5bd27654a79bf6da',
+  blockHash: '0x69304c7b6d072c157245d9f93f09661fe86d30f6befc28ba7c188ca33685d6cf',
+  blockNumber: 3574944,
+  from: '0x62dB6C4b118d7259c23692B162829E6bd5e4d5B0',
+  to: '0x0000000000000000000000000000000001000006',
+  method: '',
+  events: [
+    BridgeEvent {
+      name: 'release_request_received',
+      signature: '0x8e04e2f2c246a91202761c435d6a4971bdc7af0617f0c739d900ecd12a6d7266',
+      arguments: {
+        sender: '0x62dB6C4b118d7259c23692B162829E6bd5e4d5B0',
+        btcDestinationAddress: '0xcab5925c59a9a413f8d443000abcc5640bdf0675',
+        amount: '5130000'
+      }
+    }
+  ]
+}
+
+{
+  txHash: '0x77b15a4b54029cc9edec8096606a7b7eae1ad749c3a98ed4e0e393028e568d2f',
+  blockHash: '0x2c819a09d8e00fed67ea216c563d14e2b8286f4c0a2480b764f6dd86b3662396',
+  blockNumber: 3574946,
+  from: '0xD478f3CE39cc5957b890C09EFE709AC7d4c282F8',
+  to: '0x0000000000000000000000000000000001000006',
+  method: BridgeMethod {
+    name: 'receiveHeaders',
+    signature: '0xe5400e7b',
+    arguments: {
+      blocks: [
+        '0x0000a0200f0a949fe1d050024b2077c7528792edb620f89cb5db089f29000000000000005e54b6818e31f89e69f1ddd523b7a58d2d45259b77a110fd875b8328de2b5d1e0750e6639cde2c1983e2d76a'
+      ]
+    }
+  },
+  events: []
+}
+
+...
+```
+
+It will continue to print as it finds new Bridge methods or events in the upcoming blocks.
+
+Available param options:
+
+* [--fromblock=number] defaults to `lastest`
+* [--network=network] defaults to `testnet`
+* [--pegin] if provided, prints peg-in related transactions. It can be used along `--pegout` to only print peg-in and peg-out related transactions
+* [--pegin] if provided, prints peg-out related transactions  It can be used along `--pegin` to only print peg-in and peg-out related transactions
+* [--methods="['bridgeMethod1', 'bridgeMethod2']"] prints transaction data of calls made to any of the specified bridge methods. Defaults to all Bridge methods
```

Comment:
Yes it can. There are a couple of pegin related methods. `registerBtcCoinbaseTransaction`, `registerBtcTransaction`, `registerFastBridgeBtcTransaction`.

Reply:
Nice!